scss/_popover.scss

```
@@ -119,6 +119,12 @@
       top: var(--#{$prefix}popover-border-width);
       border-bottom-color: var(--#{$prefix}popover-bg);
     }
+
+    // `:has` is not yet supported by our `.browserlistrc` but since it adds an extra layer of specificity,
+    // it doesn't hurt the browsers that don't support it.
+    &:has(+ .popover-header)::after {
```

```suggestion
    &:has(+ .popover-header)::after,
    &:has(+ * .popover-header)::after {
```

I'd would be nice to have as a library builder. Since our library has a div there inbetween to controll the size.

see: https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/packages/bootstrap-vue-next/src/components/BPopover/BPopover.vue#L24

---

Let us think about it with @twbs/css-review, but my initial impression is that your usage seems specific to your downstream library, meaning it should handle this styling specificity independently by either:
- Adding its own overrides with `&:has(+*.popover-header)::after`
- Adjusting the DOM structure to align with our component's expected layout (probably not doable in that case on your side)

Let me walk you through my thought process. Incorporating this rule directly into Bootstrap would indeed benefit your library. However, I'm concerned that adding * selectors (or similar rules) to support various downstream libraries could lead to additional constraints. Furthermore, this rule isn't inherently tested or visible within our environment, so there's a risk it could eventually be removed as redundant, given its lack of relevance to our internal use cases.

Additionally, if you later adjust the structure and no longer need this rule, it would remain in our codebase indefinitely, serving no purpose for anyone.

---

> We implament the popover with floating-ui

Oh yeah, got it! We plan to switch to it in v6 :)